application/builtin/contract/deposit/deposit.go

```diff
@@ -52,6 +54,10 @@ type Deposit struct {
 // New creates new deposit.
 func New(txHash string, lockup int64, vesting int64, vestingStep int64) (*Deposit, error) {
 
+	if vesting%vestingStep != 0 {
+		return nil, errors.New("vesting is not multiple of vestingStep")
```

Comment:
this check is also needed in bootstrap, when vesting and vestingStep is configured

Reply:
Fixed